### PR TITLE
(DOCSP-37567) Remove Dart GraphQL test and dependency

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-37567

This is an internal change to remove the GraphQL Dart/Flutter test and package dependency. Because App Services deprecated its GraphQL API, we no longer need this test or dependency.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

- **Flutter** SDK
  - Internal.  Remove the GraphQL Dart/Flutter test and package dependency

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/dart/pubspec.lock [hunk 1]
```diff
@@ -345,14 +281,6 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "2.3.1"
-  hive:
```
Comment (lindseymoore) [inline examples/dart/pubspec.lock:348]: Were these dependencies without 'gql' related to removing graphql or something separate?

**Review by lindseymoore — COMMENTED** [review]
Just wanted to check that the dependencies in the `pubspec.lock` file without `gql` label were removed for a reason.

File: examples/dart/pubspec.lock (hunk 1 above)
Comment (krollins-mdb) [reply examples/dart/pubspec.lock:348]: The `pubspec.gql` file is auto-generated based on the dependencies listed in `pubspec.yaml`. These seemingly unrelated dependencies are "transitive" which means that they're dependencies of the dependencies in `pubspec.yaml`. They can change frequently.

**Review by lindseymoore — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**